### PR TITLE
Bump cadvisor fix, plus some friends!

### PR DESCRIPTION
**Pull request by haircommander**
https://github.com/openshift/google-cadvisor/pull/5

Signed-off-by: Peter Hunt <pehunt@redhat.com>

### CODE REVIEW

**Comment by haircommander** [comment]
@mrunalp PTAL

**Comment by mrunalp** [comment]
cadvisor bump looks fine to me :+1:
cc: @sjenning

**Comment by soltysh** [comment]
/hold
You need sign off from @deads2k since he's working on the rebase

**Comment by sjenning** [comment]
this need a rebase now that the 1.13 rebase landed
/hold cancel

**Comment by haircommander** [comment]
@sjenning @bparees @deads2k PTAL

File: glide.lock [hunk 1]
```diff
@@ -1860,7 +1860,7 @@ imports:
   - pkg/util/proto/testing
   - pkg/util/proto/validation
 - name: k8s.io/kubernetes
-  version: d8b46d72a413981e6ea99605dcef3b8ca3f2223f
+  version: 6112ef84c44b5db537e9fadffcafa06b70ef4e5b
```
Comment (bparees) [inline glide.lock]: @soltysh as usual i have to ask why we have a k8s commit changing w/o k8s content changing...

Comment (soltysh) [reply glide.lock]: @bparees we're syncing our vendored k8s out to openshift/kubernetes repo, but only contents, so the contents is in sync between the two, hashes not quite.

**Pull request closed without merging**